draft-ietf-rats-corim.md

```diff
@@ -1429,9 +1429,155 @@ is extended using Endorsements etc. from the accepted tags.
+does not match then the Reference Value does not match.
+
+The algorithm used to match the `measurement-values-map` entries
+depends on whether the reference value is tagged,
+and on the `measurement-values-map` key which identifies the entry.
+
+If the Reference Value `measurement-values-map` value starts with a CBOR tag
+then the Verifier MUST use the algorithm associated with that tag to match
+the entries.
+
+This specification defined handling for some tags, which is described in
+sub-sections below. Profiles may define algorithms describing the handling
+of additional tags.
+
+If the Verifier does not recognise the Refernce Value CBOR tag value then
+the Reference Value does not match.
+
+If the Reference Value is not tagged and the measurement-value-map key is a
+special value described in the sub-sections below,
+then the algorithm appropriate to that key is used to match the entries.
+
+If the Reference Value is not tagged, and the `measurement-values-map` key
+is not a special value described below, then the entries are compared
+using binary comparison of their CBOR encoded values. If the values
+are not binary identical then the Reference Value does not match.
+
+Note that while specifications may extend the matching semantics using tags,
+there is no way to extend the matching semantics of special key values.
+Any new keys requiring special handling must have an appropriate tag in the
+reference value.
+
+If the Reference Value contains an `authorized-by` field then the Verifier
+SHALL check which entities authorized the Accepted Claims Set entry compared
+in the steps above. If the entry was not authorized by one of the keys from
+the `authorized-by` field then the Reference Value does not match.
+
+If all checks above have been performed successfully then the Reference Value
+matches.
+
+##### Comparison for svn entries
+
+The value stored under `measurement-values-map` key 1 is a SVN, which is stored
+in the Accepted Claims Set as a UINT.
+
+If the Reference value for `measurement-values-map` key 1 is an untagged UINT or
+a UINT tagged with #6.552 then an equality comparison is performed. If the value
+of the SVN in Accepted Claims Set is not the same as the value in the Reference
+Value then the Reference Value does not match.
+
+If the Reference value for `measurement-values-map` key 1 is a UINT tagged with
+#6.553 then a minimum comparison is performed. If the value of the SVN in
+Accepted Claims Set less than the value in the Reference Value then the
+Reference Value does not match.
+
+##### Comparison for digests entries
+
+The value stored under `measurement-values-map` key 2, or a value tagged with
+#6.TBD is a digest entry. It contains one or more digests, each measuring the
+same object. A Reference Value may contain multiple digests, each with a
+different algorithm, if any of the algorithms meets the security requirements
+of the Reference Value author.
```

Comment:
```suggestion
of the Reference Value signer.
```
Note: There are two roles defined for corim: signer and manifest-creator. 'author' is ambiguous as to which role is appropriate. By default the rim signer is used for authority, but there may be a case for manifest-creator if it is supplied. Corim should document these roles better so that it is clear which entity is authoritatively supplying the measurements.

Reply:
This text was removed, but I think the entity which signed the CoRIM is irrelevant for most of the meaning of the CoRIM.